scripts/enforce-explicit-annotation-import-in-astubs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+exit_code=0
+
+for file in $(git ls-files '*.astub'); do
+  output=$(comm -23   `: print the lines present in the first output but missing from the second` \
```

Comment:
``` `: ....` ``` is a tricky way to introduce comments in shell that won't reach the end of line (and thus include the trailing `\`)